### ISSUE TITLE
[BUG] Users are unable to provision multi-node vSphere RKE1 Custom clusters

### ISSUE THREAD

**Issue by jameson-mcghee** [rancher/rancher#39361]
**Rancher Server Setup**
- Rancher version: v2.6.8, v2.6.9, and v2.7-head (Commit ID: 9f1e043)
- Installation option (Docker install/Helm Chart): Docker
   - If Helm Chart, Kubernetes Cluster and version (RKE1, RKE2, k3s, EKS, etc): N/A
- Proxy/Cert Details: N/A

**Information about the Cluster**
- Kubernetes version: v1.19.16-rancher2-1, v1.23.10-rancher1-1, v1.24.4-rancher1-1 
- Cluster Type (Local/Downstream): Downstream
   - If downstream, what type of cluster? (Custom/Imported or specify provider for Hosted/Infrastructure Provider): Custom RKE1

**User Information**
- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom): Standard User with Cluster Owner Role
  - If custom, define the set of permissions: N/A

**Describe the bug**
Users are unable to provision RKE1 Custom Clusters with more than 1 node on vSphere.

**To Reproduce**
1. In a vSphere hosted instance of Rancher, create an RKE1 custom cluster
2. Inside vSphere, spin up 8 nodes
3. Run the ETCD-only registration command on 3 of the nodes
4. Run the Control Plane-only registration command on 2 of the nodes
5. Run the Worker-only registration command on 3 of the nodes

**Result**
Note that only one of the nodes appears to register in the Rancher UI

**Expected Result**
The cluster registers and provisions all nodes successfully without error.

**Screenshots**
![image](https://user-images.githubusercontent.com/13294321/196774992-9e329303-69ea-4150-ab3b-1221da0bd1b5.png)
![image](https://user-images.githubusercontent.com/13294321/196775023-e64fe16e-4572-465c-bc8f-7aa83a8097ba.png)
![image](https://user-images.githubusercontent.com/13294321/196773575-4c7fc207-fbf7-40e7-8914-a58ec853d58f.png)
![image](https://user-images.githubusercontent.com/13294321/196773608-f2ef5460-a82c-4819-a565-5f23e5c4e028.png)

**Additional context**

**_This issue only reproduces on vSphere_**

No Provisioning Logs were available in the Rancher UI, but the following was observed in the ETCD Docker logs:
```
sudo docker logs 74e8cdfc4752 | grep error
INFO: Arguments: --server https://{HOST IP} --token REDACTED --ca-checksum 4e2eab7d222b0d953a95703d7b6b50d3bc6a425a0b101bff2c4e5246d664a2b3 --etcd
INFO: Environment: CATTLE_ADDRESS={HOST IP} CATTLE_INTERNAL_ADDRESS= CATTLE_NODE_NAME=localhost CATTLE_ROLE=,etcd CATTLE_SERVER=https://{IP ADDRESS} CATTLE_TOKEN=REDACTED
INFO: Using resolv.conf: nameserver {IP ADDRESS} options edns0 trust-ad search fremont.rancherlabs.com
WARN: Loopback address found in /etc/resolv.conf, please refer to the documentation how to configure your cluster to resolve DNS properly
INFO: https://{HOST IP}/ping is accessible
INFO: Value from https://{HOST IP}/v3/settings/cacerts is an x509 certificate
time="2022-10-19T13:15:32Z" level=error msg="Failed to connect to proxy. Response status: 400 - 400 Bad Request. Response body: Operation cannot be fulfilled on nodes.management.cattle.io \"m-421aa90e079f\": the object has been modified; please apply your changes to the latest version and try again" error="websocket: bad handshake"
time="2022-10-19T13:15:32Z" level=error msg="Remotedialer proxy error" error="websocket: bad handshake"
```

**Comment by snasovich** [comment]
Technically 1.19 is no longer supported as per the support matrix the "Lowest Version certified" for RKE1 is 1.20 - https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-8/
I understand that the UI provides an option to select 1.19 though.

This is though lower priority, milestone cleared.

**Comment by jameson-mcghee** [comment]
I have confirmed that the test results observed previously was due to the use of vSphere OS template that were incompatible with the creation of RKE1 Custom Clusters, and therefore my observed test results were invalid. I did confirm that when using a valid OS template, this issue does not reproduce on v2.6.8 (see Screenshot). Therefore I am closing this ticket.

![image](https://user-images.githubusercontent.com/13294321/197063614-0f179769-cffc-43cd-a7fb-8388add473ca.png)
